pandas/tests/frame/test_dtypes.py

```diff
@@ -850,6 +850,24 @@ def test_arg_for_errors_in_astype(self):
 
         df.astype(np.int8, errors='ignore')
 
+    def test_arg_for_errors_in_astype_dictlist(self):
+        data_df = pd.DataFrame([{'col_a': '1', 'col_b': '16.5%',
+                                 'col_c': 'test'},
+                                {'col_a': '2.2', 'col_b': '15.3',
+                                 'col_c': 'another_test'}
+                                ])
+        reference_df = pd.DataFrame([{'col_a': '1', 'col_b': '16.5%',
+                                     'col_c': 'test'},
+                                     {'col_a': '2.2', 'col_b': '15.3',
+                                     'col_c': 'another_test'}
+                                     ], dtype='float64')
+        type_dict = {'col_a': 'float64', 'col_b': 'float64', 'col_c': 'object'}
+
+        with not_raises(ValueError):
+            df_astype = data_df.astype(dtype=type_dict, errors='ignore')
```

Comment:
Call this result, and probably get rid of the context manager

Reply:
I guess I feel like that leaves us right back at 11e5d421c7e9bc207f40c8057e3f0c9016a245a0 which was my first write up for a test.

Reply:
Done in 1c35cc70108a0407d964c26b5cfc07d654dd4ebc